mlflow/pytorch/__init__.py

```diff
@@ -128,6 +128,13 @@ def log_model(
     """
     Log a PyTorch model as an MLflow artifact for the current run.
 
+        .. warning::
+
+            Log the model with signature to avoid inference errors. Pytorch float precision default
+            is float32, while numpy float precision default is float64. Adding the signature will
```

Comment:
I would maybe re-frame this a bit and say:

For models without signatures, the MLflow Model Server relies on the default inferred data type from NumPy. However, PyTorch often expects different defaults, particularly when parsing floats.